sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/LanguageBatchInput.java

```diff
@@ -38,4 +39,11 @@ public LanguageBatchInput setDocuments(List<LanguageInput> documents) {
         this.documents = documents;
         return this;
     }
+
+    @Override
+    public String toString() {
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
Don't want to change auto-gene code. Using list.stream().collect(Collectors.joining(","));